### ISSUE TITLE
windows or macos trackpad scroll bug 

### ISSUE THREAD

**Issue by huangye0** [bosskmk/pluto_grid#652]
i meet this problem too.
windows can't scroll vertical or horizontal when using trackpad,but scroll using mouse is fine.
macos meets the same problem too

**Comment by huangye0** [comment]
confirm same issue : https://github.com/bosskmk/pluto_grid/issues/600

**Comment by bosskmk** [comment]
I don't have a laptop, so I can't test it in a trackpad environment, so I don't think I can solve this problem right away.
I'll try to fix the problem with a trackpad environment in the future.

**Comment by thegioiso** [comment]
@bosskmk @huangye0  maybe this will fix this issue:

create a ScrollBehavior class:
```
class CustomScrollBehavior extends MaterialScrollBehavior {
  @override
  Set<PointerDeviceKind> get dragDevices => {
        PointerDeviceKind.touch,
        PointerDeviceKind.mouse,
        PointerDeviceKind.trackpad
      };
}
```
add it to MyApp:
```
class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return FluentApp.router(
      scrollBehavior: WebScrollBehavior(),
      debugShowCheckedModeBanner: false,
  }
}
```

**Comment by bosskmk** [comment]
@thegioiso 
If so, let's distribute a modified version so that the `scrollBehavior` that is forced inside `PlutoGrid` can be customized.
After distribution, we will inform you here.
At that time, please test in the trackpad environment. (I don't have a laptop)
Thank you.

**Comment by bosskmk** [comment]
The problem has been fixed in PlutoGrid version 5.4.4.
Problem caused by trackpad not being set in dragDevices.

https://pluto.weblaze.dev/scrollbar-and-scroll-behavior

@thegioiso 
thank you.
